.github/workflows/update-snapshots.yml

```diff
@@ -85,9 +88,6 @@ jobs:
       - name: Deploy the backend
         uses: ./.github/actions/deploy-backend
 
-      - name: Prepare
-        uses: ./.github/actions/prepare
-
```

Comment:
The prepare action is already called by the deploy-backend action in the previous step.  No need to do it twice.